docs/content/testing/running-tests.md

```diff
@@ -0,0 +1,62 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Running tests
+
+Run the following commands in your terminal from the root of the repository.
+
+For the first run, you need these commands:
+
+```bash
+./prepare-workspace.sh
+./ci/build-test.sh
+```
```

Comment:
* If I have to run these, is there any way to know what success or failure looks like or are they silent in both instances? Or is neither success or failure an option, they just run?

Reply:
Good point! I'll add snippets on what good/bad looks like :)